package.json

```diff
@@ -105,7 +105,7 @@
     "copy:types": "grunt copy:types",
     "build:types": "yarn clean:types && yarn generate:types && yarn copy:types",
     "start:mock-server": "MOCK_SERVER_PORT=3030 nodemon --delay 500ms -e js,json --watch playground --exec 'node playground/mocks/server.js'",
-    "test:parity-setup": "mkdir -p test-reports/testcafe && yarn generate-config && yarn workspace v3 build && yarn workspace v3 serve",
+    "test:parity-setup": "mkdir -p test-reports/testcafe && yarn generate-config && mockDuo=true yarn workspace v3 build && yarn workspace v3 serve",
```

Comment:
need to add `mockDuo` flag so the mock duo response will be used in the bacon run